tests/Functional/DifferentBaseCacheDirTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Crunz\Tests\Functional;
+
+use Crunz\Application;
+use PHPUnit\Framework\TestCase;
+
+class DifferentBaseCacheDirTest extends TestCase
+{
+    /**
+     * @test
+     *
+     * @runInSeparateProcess
+     */
+    public function different_base_cache_dir_is_used(): void
```

Comment:
Actually this test do not check that setting `CRUNZ_BASE_CACHE_PATH` will change cache directory or am I missing something?

Reply:
You are right, this only tests, that the path givein in the `Application` constructor is created. Depending on the suggested change above, I will add another test.

Reply:
I changed this test to use the env.